added chars to template

jamessimo opened the pull request:
Added transparent chars to fontawesome-style.html (using css) to simulate the ability to copy and paste icons from the sample.html page into sketch.

Wrapped the chars in a span which are transparent and absolutely positioned inside the icon 
```
 <li>
````
tag inside fontawesome-style.html, this doesn't change the look of the rendered sample.html page, just allows users to highlight and copy icons into sketch

cognitom commented:
@jamessimo cool! I like it.
Could you add the same to `fundation-style.html`, too?

jamessimo commented:
@cognitom added in  f7967e3b9a1c50fea8bec7b812e384dd7efe6da5 👍

cognitom commented:
Oops, you've added the line to the same file. I mean, there're two templates, so add the same functionality to both of them:

- fontawesome-style.html
- fundation-style.html

Sorry if my direction was not clear...

jamessimo commented:
Fixed, sorry you were clear. I just applied the foundation changes to font awesome and committed without checking properly.

I reverted the FA to my original commit and applied the changes to foundation.

cognitom commented:
@jamessimo no prob, thank you so much!
